tests/conftest.py

```diff
@@ -48,3 +50,137 @@ def refDir():
+def transactionResponseDelete():
+    id = 'S1A_EW_GRDM_1SDH_20200420T023244_20200420T023348_032205_03B97F_72EB'
+    text = ('<?xml version="1.0" encoding="UTF-8" standalone="no"?>'
+            '<!-- pycsw 2.7.dev0 -->'
+            '<csw:TransactionResponse xmlns:csw="http://www.opengis.net/cat/csw/2.0.2" '
+            'xmlns:dc="http://purl.org/dc/elements/1.1/" '
+            'xmlns:dct="http://purl.org/dc/terms/" '
+            'xmlns:gmd="http://www.isotc211.org/2005/gmd" '
+            'xmlns:gml="http://www.opengis.net/gml" xmlns:ows="http://www.opengis.net/ows" '
+            'xmlns:xs="http://www.w3.org/2001/XMLSchema" '
+            'xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.0.2" '
+            'xsi:schemaLocation="http://www.opengis.net/cat/csw/2.0.2 '
+            'http://schemas.opengis.net/csw/2.0.2/CSW-publication.xsd">'
+            '<csw:TransactionSummary>'
+            '<csw:totalInserted>0</csw:totalInserted>'
+            '<csw:totalUpdated>0</csw:totalUpdated>'
+            '<csw:totalDeleted>1</csw:totalDeleted>'
+            '</csw:TransactionSummary>'
+            '</csw:TransactionResponse>')
+    return [id, text]
+
+@pytest.fixture(scope='session')
+def transactionResponseDeleteFails():
+    id = 'S1A_EW_GRDM_1SDH_20200420T023244_20200420T023348_032205_03B97F_72EB'
+    text = ('<?xml version="1.0" encoding="UTF-8" standalone="no"?>'
+            '<!-- pycsw 2.7.dev0 -->'
+            '<csw:TransactionResponse xmlns:csw="http://www.opengis.net/cat/csw/2.0.2" '
+            'xmlns:dc="http://purl.org/dc/elements/1.1/" '
+            'xmlns:dct="http://purl.org/dc/terms/" '
+            'xmlns:gmd="http://www.isotc211.org/2005/gmd" '
+            'xmlns:gml="http://www.opengis.net/gml" xmlns:ows="http://www.opengis.net/ows" '
+            'xmlns:xs="http://www.w3.org/2001/XMLSchema" '
+            'xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" version="2.0.2" '
+            'xsi:schemaLocation="http://www.opengis.net/cat/csw/2.0.2 '
+            'http://schemas.opengis.net/csw/2.0.2/CSW-publication.xsd">'
+            '<csw:TransactionSummary>'
+            '<csw:totalInserted>0</csw:totalInserted>'
+            '<csw:totalUpdated>0</csw:totalUpdated>'
+            '<csw:totalDeleted>0</csw:totalDeleted>'
+            '</csw:TransactionSummary>'
+            '</csw:TransactionResponse>')
+    return [id, text]
+
+@pytest.fixture(scope='session')
+def transactionResponseDeleteUnknownTagName():
+    id = 'S1A_EW_GRDM_1SDH_20200420T023244_20200420T023348_032205_03B97F_72EB'
+    text = ('<?xml version="1.0" encoding="UTF-8" standalone="no"?>'
+            '<!-- pycsw 2.7.dev0 -->'
+            '<SomeNewTagName>'
+            '<TransactionSummary>'
+            '<totalInserted>0</totalInserted>'
+            '<totalUpdated>0</totalUpdated>'
+            '<totalDeleted>1</totalDeleted>'
+            '</TransactionSummary>'
+            '</SomeNewTagName>')
+    return [id, text]
```

Comment:
I think it is better if these are defined in the pycsw test file, as they are specific to that module?

If so, the scope should be changed to "module"

Reply:
Done in 069863dcb3295e17eab877e60c9c4b34692ed20d